resources/lang/fr/dashboard.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
 return [
-    'totalhours'     => 'Heures Totales',
-    'yourbalance'    => 'Votre Solde',
-    'yourlastreport' => 'Votre dernier Rapport',
-    'noreportsyet'   => 'Aucun Rapport',
+    'totalhours'     => 'Total heures',
```

Comment:
I think "heures totales" is more french than "total heures"

Reply:
I prefer "total heures" because it's more a tittle of a table. (my feeling).
No problem for me to use hte one you prefer.

Reply:
Yea both work I think

Reply:
total heures me plait mieux, perso

Reply:
Jsais pas trop, c'est pour le titre sur le dashboard donc c'est un peu le nombre d'heures totales que tu as fais pour la va qui est affiché à côté